src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -111,13 +117,30 @@ public static File renderTileMetricsFileFromBasecallingDirectory(final File illu
                 final int phasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PHASING_BASE);
                 final int prePhasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PREPHASING_BASE);
 
+                float phasingValue = 0;
+                float prePhasingValue = 0;
+
                 if (!(codeMetricsMap.containsKey(phasingCode) && codeMetricsMap.containsKey(prePhasingCode))) {
-                    throw new PicardException("Don't have both phasing and prephasing values for tile");
+                    final String message = String.format(
+                            "Don't have both phasing and prephasing values for read %s cycle %s.  Phasing code was %d and prephasing code was %d.",
+                            isFirstRead ? "one" : "two", descriptorIndex+1, phasingCode, prePhasingCode
+                    );
+                    switch (validationStringency) {
+                        case LENIENT:
+                            LOG.warn(message);
```

Comment:
so in this case, you'll get 0 for both phasing and pre-phasing in the case that one of them is missing. 

1. do you understand _why_ one of them could be missing? a comment would be helpful for future improvement, even if you are not going to do it yourself.
2. if one is available (of phasing and pre-phasing) do you not want to grab that one?